Documentation/ApiOverview/UpdateWizards/Concept/Index.rst

```diff
@@ -0,0 +1,33 @@
+.. include:: ../../../Includes.txt
+
+.. _update-wizards-concept:
+
+=============================
+The concept of update wizards
+=============================
+
+Update wizards are single PHP Classes which provide an automated way to update certain
+parts of a TYPO3 installation. There parts might be database or file system related.
+
+Wizards should be provided to ease updates for integrators and administrators. They
+are an addition to the database migration which is handled by the Core based on
```

Comment:
"[...] database migration, which [...]"

* comma before which (should be consistent in the whole docu...)